src/components/BooksDisplay.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import AddBook from './AddBook';
+
+const BooksDisplay = () => (
+  <>
+    <section>
+      <ul className="books-to-read">
+        <li>
+          <span>BookState</span>
+          <button type="button" className="delete-btn">Delete</button>
+        </li>
+      </ul>
+    </section>
+    <AddBook />
+  </>
```

Comment:
Hi,

Thank you for submitting your code for review. I have reviewed your code and found some issues that need to be fixed before your code can meet the project requirements. Here are my comments and suggestions:

- Please move the list of books from your `<BooksDisplay>` component to your `<BookList>` component. Your `<Books>` component should only render the `<BookList>` component and the `<AddBook />` component. This will match the project requirement of creating a `<Books>` component that contains other components.
- Please create a `<BookList>` component that takes the book data as a prop and maps over it to render each book using the `<Book>` component. This will match the project requirement of creating a `<BookList>` component that renders a list of books using the `<Book>` component.
- Please create a `<Book>` component that takes the book data as a prop and renders the title and the author of the book. Please also add a button for removing books inside the `<Book>` component, even if it does not work yet. This will match the project requirement of creating a `<Book>` component that renders the book details and has a button for removing books.

Please make these changes and resubmit your code for review. If you have any questions or need any help, please let me know.

Happy coding!